src/server/routes/forwardingApp.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2018-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { taxonomyLookup } from '../helpers/taxonomyLookup';
+import { getToken } from '../helpers/auth';
+import { storeAccessToken } from '../../util/apiHelpers';
+
+export async function forwardingApp(req, resp, next) {
```

Comment:
Maybe rename to `forwardRoute` or something similar

Reply:
Agreed